File: android/app/src/main/kotlin/vn/netacom/lomo/tracking/FacebookTracking.kt
package vn.netacom.lomo.tracking

import android.content.Context
import com.facebook.appevents.AppEventsLogger

class FacebookTracking {
    lateinit var logger:AppEventsLogger

    companion object {
        var instance = FacebookTracking()
    }

    fun init(context: Context){
         logger = AppEventsLogger.newLogger(context)
    }

    fun tracking(event:String){
        logger.logEvent(event)
    }
}

File: ios/Runner/tracking/FacebookTracking.swift
//
//  FacebookTracking.swift
//  Runner
//
//  Created by Meo Luoi on 20/08/2021.
//

import Foundation
import FBSDKCoreKit

class FacebookTracking{
    static let instance = FacebookTracking()
    
    func tracking(event:String){
        AppEvents.logEvent(AppEvents.Name(event))
    }
}
